pyoozie/model.py

```diff
@@ -159,38 +170,37 @@ def suspended(cls):
             return [status for status in cls if status.is_suspended()]
 
         def is_unknown(self):
-            return self.value.status_id == 0
+            return self._value_.status_id == 0
```

Comment:
Linter won't accept `self.value` here in python3 -- but seems content with `_value_`. WTF?